chia/consensus/block_body_validation.py

```diff
@@ -46,9 +52,10 @@ async def validate_block_body(
     peak: Optional[BlockRecord],
     block: Union[FullBlock, UnfinishedBlock],
     height: uint32,
-    cached_cost_result: Optional[CostResult] = None,
-    fork_point_with_peak: Optional[uint32] = None,
-) -> Tuple[Optional[Err], Optional[CostResult]]:
+    npc_result: Optional[NPCResult],
```

Comment:
Why is this optional? Later there is `assert npc_result is not None`

Reply:
It's only there for transaction blocks. Ill add an assert that it's False for non tx blocks